stream.go

```diff
@@ -121,9 +123,23 @@ func (s *Stream) WriteShort(status int64, data1 int64, data2 int64) error {
 	return s.Write([]Event{evt})
 }
 
-// Writes a system exclusive MIDI message to the output stream.
+// WriteSysExBytes writes a system exclusive MIDI message given as []byte to the output stream.
```

Comment:
Shouldn't it be message given as a []byte to the output stream? Hence it is pronounced as "message given as byte slice to the output stream."